streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -125,16 +123,16 @@ public void close() {
 
     @Override
     public long approximateNumEntries() {
-        return inner.approximateNumEntries();
+        return wrappedStore().approximateNumEntries();
```

Comment:
This is ugly -- we should have access to the wrapped store directly via inheritance. (also below)